examples/kitty/kitty.mk

```diff
@@ -145,28 +146,17 @@ vmm.elf: ${VMM_OBJS} libvmm.a
 # Build with two threads in parallel
 nproc=2
 
-
-micropython.elf: mpy-cross libsddf_util_debug.a libco.a config.py manifest.py \
-		kitty.py pn532.py font_height50.py font_height35.py writer.py \
-		$(LIONSOS)/dep/libmicrokitco/Makefile
-	make  -C $(LIONSOS)/components/micropython -j$(nproc) \
-			MICROKIT_SDK=$(MICROKIT_SDK) \
-			MICROKIT_BOARD=$(MICROKIT_BOARD) \
-			MICROKIT_CONFIG=$(MICROKIT_CONFIG) \
-			MICROPY_MPYCROSS=$(abspath mpy_cross/mpy-cross) \
-			MICROPY_MPYCROSS_DEPENDENCY=$(abspath mpy_cross/mpy-cross) \
-			BUILD=$(abspath $(BUILD_DIR)) \
-			LIBMATH=${LIBMATH} \
-			FROZEN_MANIFEST=$(abspath manifest.py) \
-			CONFIG_INCLUDE=$(abspath $(CONFIG_INCLUDE)) \
-			ENABLE_I2C=1 \
-			ENABLE_FRAMEBUFFER=1 \
-			V=1
+MICROPYTHON_LIBMATH := ${LIBMATH}
+MICROPYTHON_CONFIG_INCLUDE := $(CONFIG_INCLUDE)
+MICROPYTHON_FROZEN_MANIFEST := manifest.py
+MICROPYTHON_ENABLE_I2C := 1
+MICROPYTHON_ENABLE_FRAMEBUFFER := 1
+include $(MICROPYTHON)/micropython.mk
 
 config.py: ${KITTY_DIR}/board/$(MICROKIT_BOARD)/config.py
 	cp $< $@
 
-manifest.py: ${KITTY_DIR}/manifest.py
+manifest.py: ${KITTY_DIR}/manifest.py kitty.py pn532.py font_height50.py font_height35.py writer.py config.py
 	cp $< $@
```

Comment:
manifest.py isn't changed if kitty.py etc., change --- they're not dependencies.
The cp line here will copy the most recent of them to `manifest.py` which is not what you want.

Reply:
Yes you're right.